test/tracer_options_test.cpp

```diff
@@ -61,45 +78,52 @@ TEST_CASE("tracer options from environment variables") {
         {"DD_TRACE_REPORT_HOSTNAME", "true"},
         {"DD_TRACE_ANALYTICS_ENABLED", "true"},
         {"DD_TRACE_ANALYTICS_SAMPLE_RATE", "0.5"},
-        {"DD_TAGS", "host:my-host-name,region:us-east-1,datacenter:us,partition:5"}},
+        {"DD_TAGS", "host:my-host-name,region:us-east-1,datacenter:us,partition:5"},
+        {"DD_TRACE_RATE_LIMIT", "200"},
+        {"DD_TRACE_SAMPLE_RATE", "0.7"}},
```

Comment:
Also just curious: should we test this for different locales (such as ones that use comma instead of period), or would that just be unit-testing`std::stod`?

Reply:
It's a good question.  `std::stod` does what `std::strtod` does, which does what `strtod`, which changes based on the C locale.

My preference is to not think about it, and to hope that the tests never have to run in a different locale. I do wonder about people pulling our code and running the tests, though. Hm.

One option is to interpolate always from a `double` serialized using `to_string`.

Let's leave it like this for now.